bokeh/core/templates.py

```diff
@@ -35,6 +35,8 @@
 
 FILE = _env.get_template("file.html")
 
+ZEPPELIN_FILE = _env.get_template("zeppelin.html")
+
 NOTEBOOK_LOAD = _env.get_template("notebook_load.html")
```

Comment:
If we are going do the "one notebook type" to "N notebook types" then we need to make things explicit. I think the new template should be `ZEPPELIN_NOTEBOOK_LOAD` and then the old `NOTEBOOK_LOAD` should be renamed to `JUPYTER_NOTEBOOK_LOAD`

This principle holds in general, and will make this PR necessarily be somewhat bigger. Basically, if we are to support two notebook types, it's not good to have one thing just called "notebook" (which is implicitly jupyter) and then a new thing called "zeppelin notebook". We need to make sure *everywhere* we explicitly distinguish "jupyter notebook" and "zeppelin notebook". So e.g. there is also `bokeh.util.notebook` that has functions like `load_notebook` in it that will need to be renamed to something like `load_jupyter`. Then, in order to be user-friendly a function `load_notebook` should be added that prints a deprecation warning, and defers to `load_jupyter`. There will need to be migration notes about all these changes in the release notes.